multigeiger/multigeiger.ino

```diff
@@ -137,6 +137,9 @@ enum {SEND_CPM,SEND_BME};
 // Measurement interval (default 2.5min) [sec]
 #define MEASUREMENT_INTERVAL 150
 
+// How many HV capacitor charge pulses to generate before giving up:
+#define MAX_CHARGEPULSES 1000
```

Comment:
btw, i noticed that with the current value of 1000, charging will take up to ~2.6s which is way too long considering we recharge in the main loop every 1000ms anyway, see:
```
  // Pulse the high voltage if we got enough GMC pulses to update the display or at least every 1000ms.
```